Frontend/ui-driver/src/Flow.purs

```diff
@@ -206,7 +207,7 @@ getDriverInfoFlow = do
           pure unit
         setValueToLocalStore IS_DRIVER_ENABLED "true"
         let (Vehicle linkedVehicle) = (fromMaybe dummyVehicleObject getDriverInfoResp.linkedVehicle)
-        modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen { data = homeScreen.data {driverName = getDriverInfoResp.firstName, vehicleType = linkedVehicle.variant ,  driverAlternateMobile =getDriverInfoResp.alternateNumber   }
+        modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen { data = homeScreen.data {driverName = getDriverInfoResp.firstName, vehicleType = linkedVehicle.variant ,  driverAlternateMobile =getDriverInfoResp.alternateNumber,base64Image = (fromMaybe "" getDriverInfoResp.mediaUrl)   }
```

Comment:
Don't use fromMaybe, Handle it if it is Nothing

Reply:
it will assign it an empty string if Nothing